### PR TITLE
Travis: build on OS X (take 2)

### PR DESCRIPTION
**Pull request by larsrh**
Supersedes #714.

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -1,13 +1,19 @@
-language: scala
+language: java
```
Comment (muuki88) [inline .travis.yml]: why is this change necessary?

Comment (larsrh) [inline .travis.yml]: The `sbt` launcher script is unavailable on OS X anyway.

Comment (larsrh) [inline .travis.yml]: (The other option would be to not include it in the repository but fetch it in `before_install`.)

**Comment by muuki88** [comment]
Having a dedicated launcher script is worth it, when we have OS X builds now. Thanks a lot for your effort :)

**Comment by lunaryorn** [comment]
@larsrh Psst, wise man use force push, young padawan :sunglasses:  :stuck_out_tongue_winking_eye:
